src/main/java/org/opentripplanner/raptor/api/response/RaptorResponse.java

```diff
@@ -69,12 +62,20 @@ public RaptorRequest<T> requestUsed() {
     return requestUsed;
   }
 
+  /**
+   * Return {@code true} if the heuristic and the main search does not find any connections.
+   * Searching again with another time/search-window will not produce any results. There is no paths
+   * in the set of days provided in the transit data with the request usd.
```

Comment:
Maybe I don't get it but doesn't this do contradict the documentation of the error type?

Reply:
The doc on `noTransitConnectionInSearchWindow` is changed in this PR, but this is mapped to `noTransitConnection`.

>  `noTransitConnection` - "No transit connection was found between the origin and destination withing the operating day or the next day"
>  
>  `noTransitConnectionInSearchWindow`- "Transit connection was found, but it was outside the search window, see metadata for the next search window"

Reply:
To be clear, we return `noTransitConnection` when you should not page and `noTransitConnectionInSearchWindow` when you should.